support/routes/v2/isobot/index.ts

```diff
@@ -41,5 +42,26 @@ const handleWhitelistEmails: RequestHandler<
   }
 }
 
+const handleDnsChecker: RequestHandler<
+  never,
+  DnsCheckerResponse | { message: string },
+  SlackPayload,
+  unknown,
+  never
+> = async (req, res) => {
+  const validatedDomain = botService.getValidatedDomain(req.body)
+  if (!validatedDomain)
+    return res.status(200).send({
+      message: `Sorry, \`${req.body.text}\` is not a valid domain name. Please try again with a valid one instead.`,
+    })
+
+  // We need to return 200 OK to Slack before we process the request
```

Comment:
is this comment misplaced? seems like we only return a 200 after we actually to the dns calls and get a proper response. 

also why are we optimising for api response times here ah? shouldn't this just be a ping to the domain name, should be at most 2-3 network calls no?

Reply:
Oh yeah the comment was for a previous iteration where the response was done later, but opted to return immediately for simplicity.

Removed in 15d5512dc78e9bce861968a01f86caf51172741b!